specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -719,6 +719,35 @@
         "location"
       ]
     },
+    "OrcDataset": {
```

Comment:
It looks that your new definitions haven't been used in your specification and your path is {}. Is there any story for that?

Reply:
We use "x-ms-discriminator-value" here to extend the "Dataset" by add a new type "Orc", and the "Dataset" has already referenced in API request at specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json. So the newly added one can be used in API request.

Reply:
Right. LGTM from API specification.